pkg/ebpf/c/tracee.bpf.c

```diff
@@ -2440,23 +2447,22 @@ int tracepoint__sched__sched_process_fork(struct bpf_raw_tracepoint_args *ctx)
         bpf_map_update_elem(&new_pids_map, &child_pid, &child_pid, BPF_ANY);
     }
 
-    if (event_chosen(SCHED_PROCESS_FORK)) {
```

Comment:
I think we should keep this gate to avoid flooding userspace with process info events.
We can guard with a new config_option in addition to the existing SCHED_PROCESS_FORK, 
WDYT?